exercises/accumulate/accumulate.scm

```diff
@@ -0,0 +1,2 @@
+(define accumulate
```

Comment:
The skeleton should be a function having same argument names as the example, `f` and `xs`